### ISSUE TITLE
Renamed items are counted many times

### ISSUE THREAD

**Issue by GoogleCodeExporter** [waterxi/svnplot#33]
```
You can see this issue in the "directory size" graph for example.

```

Original issue reported on code.google.com by `cyber.s...@gmail.com` on 28 Feb 2010 at 6:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `nitinbh...@gmail.com` on 12 Mar 2010 at 4:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `nitinbh...@gmail.com` on 12 Mar 2010 at 5:02

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.6
```

Original comment by `nitinbh...@gmail.com` on 5 Apr 2010 at 3:26
- Changed state: **Fixed**
